mantidproject/mantid
# Imaging GUI: honor the option to change the remote scripts path

Fixes this issue that was found while trying to develop/debug other issues.

**To test:**
- Open the custom interface "Diffraction" => "Tomographic reconstruction"
- To test this you need an account on SCARF.  In the tab system settings, change the option The "Paths on the remote (SCARF) file system" -> "base path for reconstruction tools/scripts". Then, submit a job and check that the modified path is used.
- Also to test: after changing that option, close and open again the interface. The option should be remembered correctly (I found that this was also broken). As this uses QSettings it unfortunately escapes our current automated unit testing infrastructure.
- Check code changes.

Fixes #17233 .

_Does not need to be in the release notes._, as this an option that users will rarely change.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
